np11.py

```diff
@@ -0,0 +1,16 @@
+import numpy as np
+
+
+def diagonal_array():
+    """
+    Function that create a 3-D array with ones on a diagonal and zeros elsewhere
+    """
+    array = np.zeros((3, 3))
+    np.fill_diagonal(array, 1, wrap=True)
+    return array
+
+
+if __name__ == '__main__':
+    print(diagonal_array())
```

Comment:
This should not be here.
Don’t use git add -a ever, use git add -p
https://git-scm.com/docs/git-add